Library/Formula/gpg-agent.rb

```diff
@@ -33,9 +33,46 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+      To replace ssh-agent with gpg-agent, you need to:
+         * ensure that "enable-ssh-support" is in ~/.gnupg/gpg-agent.conf
+         * ensure that "pinentry" in ~/.gnupg/gpg-agent.conf has a graphical PIN entry
+           such as pinentry-mac (needed for GUI programs)
+         * disable the system ssh-agent with
+             launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist
+         * have launchd start gpg-agent at login
+    EOS
+  end
```

Comment:
I included the note because it's not obvious how to replace the system ssh-agent with gpg-agent.  I suppose I don't need to include the note about the enable-ssh-agent in ~/.gnupg/gpg-agent.ocnf, but if we include information about installing the plist (which happens automatically), I thought it wise to mention how to remove the "conflicting" plist from ssh-agent.

But, if you'd prefer it removed, I can definitely do that.